notebook/static/notebook/js/quickhelp.js

```diff
@@ -64,8 +71,11 @@ define([
         { shortcut: cmd_ctrl + "[",   help:i18n.msg._("dedent")  },
         { shortcut: cmd_ctrl + "a",   help:i18n.msg._("select all")  },
         { shortcut: cmd_ctrl + "z",   help:i18n.msg._("undo")  },
-        { shortcut: cmd_ctrl + "Shift-z",   help:i18n.msg._("redo")  },
-        { shortcut: cmd_ctrl + "y",   help:i18n.msg._("redo")  },
+        { shortcut: cmd_ctrl + "/",   help:i18n.msg._("comment")  },
+        { shortcut: "esc",   help:i18n.msg._("single selection")  },
```

Comment:
I'd leave this out as Esc normally switches to command mode, so it's confusing. If it stays in, let's capitalise it for consistency with the other shortcuts.